wagtailio/core/models/pages.py

```diff
@@ -9,6 +9,7 @@
 
 
 class HomePage(Page, SocialMediaMixin, CrossPageMixin):
+    template = "patterns/pages/home/home_page.html"
```

Comment:
Shall we remove [the old homepage template](https://github.com/wagtail/wagtail.org/blob/72-homepage-hero/wagtailio/core/templates/core/home_page.html)?

Reply:
yes please. let's tidy up as we go :P